x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/job_details_step/components/groups/groups_input.tsx

```diff
@@ -77,6 +77,7 @@ export const GroupsInput: FC = () => {
         onCreateOption={onCreateGroup}
         isClearable={true}
         isInvalid={validation.valid === false}
+        data-test-subj="mlJobWizardComboboxJobGroups"
```

Comment:
Nit - you may want to switch the label to `mlJobWizardComboBoxJobGroups` to match the case in `EuiComboBox`

Reply:
Done